### PR TITLE
add templates (fixes #37)

### LINKED ISSUE #37: Issue and PR templates
https://help.github.com/en/articles/about-issue-and-pull-request-templates

### PR DESCRIPTION
**Pull request by Thais-ra**
Add templates of pull request and creation of issues.

### CODE REVIEW

**Comment by CalebProvost** [comment]
Thank you @Thais-ra!
I'll point the rest of our members in our group to review it.
Feel free to [look us over](https://treehouses.io/#!pages/vi/firststeps.md) and see if you'd like to join the rest of us Virtual Interns. We could always benefit from someone with your skills.
Just drop in on our [gitter chatroom](https://gitter.im/treehouses/Lobby) and say hello.

Additionally, could you put this on a new branch for this pull request?

For the rest of the interns, the githack link is:
https://raw.githack.com/Thais-ra/treehouses.github.io/master/#!Templates/ISSUE_TEMPLATE.md
and
https://raw.githack.com/Thais-ra/treehouses.github.io/master/#!Templates/PULL_REQUEST_TEMPLATE.md

**Review by pattanawadee88 — CHANGES_REQUESTED** [review]
For PULL_REQUEST_TEMPLATE.md, please also include the raw.githack link. :)

File: Templates/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Description
+
+Concise description of what was done
+
+## Solve (Issues)
```
Comment (pattanawadee88) [inline Templates/PULL_REQUEST_TEMPLATE.md]: I think this line should be (fixes #issue_number) and this should be at the end of pull request title

**Comment by dogi** [comment]
https://github.com/open-learning-exchange/open-learning-exchange.github.io/blob/master/.github/ISSUE_TEMPLATE.md

**Comment by kjong** [comment]
I've updated the templates. Requesting reviews.

**Review by vmnet8 — APPROVED** [review]
good

**Comment by dogi** [comment]
@kjong can you also fix PR template?

**Comment by kjong** [comment]
Are there any particular changes you are looking for?
I modeled it after the one at: https://github.com/open-learning-exchange/open-learning-exchange.github.io/blob/master/.github/PULL_REQUEST_TEMPLATE.md

**Review by pattanawadee88 — APPROVED** [review]

**Review by CalebProvost — CHANGES_REQUESTED** [review]
To make things just a little bit easier, have the raw.githack link example in the PR template:
`https://raw.githack.com/UserName/Repository/Branch/Path/To/File.md`

**Review by CalebProvost — APPROVED** [review]

**Pull request merged**